grafana/overlays/nerc-ocp-obs/grafanadatasources/observability-metrics.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: observability-metrics
+  namespace: grafana
+  labels:
+    app.kubernetes.io/instance: grafana-infra
```

Comment:
We'll want to change `infra` to `obs` here I think.

Reply:
thanks, yes, i created this draft to work with Jeet on it together.

Reply:
```suggestion
    app.kubernetes.io/instance: grafana-obs
```